### ISSUE TITLE
Incompatible Plug-inc Disabled

### ISSUE THREAD

**Issue by archodessa** [robertklep/quotefixformac#37]
![screen shot 2014-11-19 at 09 05 19](https://cloud.githubusercontent.com/assets/9437256/5101946/282ed710-6fcb-11e4-99b9-960a0cc4ab04.png)

**Comment by archodessa** [comment]
There is a problem with last OS again.

**Comment by alvarnell** [comment]
No problems for me.

Are you using 
v2.6.0-beta.6 <https://github.com/robertklep/quotefixformac/releases/tag/v2.6.0-beta.6>?
-Al-

**Comment by archodessa** [comment]
Yes, of course.
![screen shot 2015-02-06 at 14 41 21](https://cloud.githubusercontent.com/assets/9437256/6080000/690fa5e2-ae15-11e4-9800-da25d88e52f4.png)

**Comment by robertklep** [comment]
@archodessa that doesn't make sense: you're saying that the plug-in won't load on 10.10.2, yet you're showing a screenshot of the plug-in?

**Comment by archodessa** [comment]
No, the problem, that it show a window and only after its closing, the Mail is running.

So, again:
1. This window;
2. Close the window;
3. Apple Mail.

Instead of:
1. Apple Mail.

Do you understand?

**Comment by archodessa** [comment]
![screen shot 2015-02-06 at 13 05 30](https://cloud.githubusercontent.com/assets/9437256/6080153/c763d32e-ae16-11e4-894f-07f6f7a4bf08.png)

**Comment by robertklep** [comment]
@archodessa yes, I understand that. But where is the screenshot coming from that is showing which version of QF is installed? Because that can only be shown when the plug-in is loaded correctly.

**Comment by alvarnell** [comment]
Sorry, but after trying 1-2-3, I still do not get your results with Mail 8.2.

-Al-

**Comment by archodessa** [comment]
@robertklep at the last current version: v2.6.0

**Comment by archodessa** [comment]
See this problem every day... ;-(

**Comment by robertklep** [comment]
@archodessa again, it doesn't make sense if you can open the QF preferences in Mail *and* get the "Incompatible Plug-Ins Disabled" message (because you can only get to the preferences if the plug-in was loaded into Mail properly).
